lib/init/rc.boot

```diff
@@ -167,6 +171,14 @@ main() {
     command -v udevd >/dev/null &&
         udevadm control --exit
 
+    log "Running rc.d scripts..."; {
+        set +f
```

Comment:
Both `set` calls can be removed as I've changed the script to only disable globbing where needed.

Reply:
ok thanks